packages/contentstack-auth/messages/index.json

```diff
@@ -2,7 +2,7 @@
   "CLI_AUTH_LOGIN_ENTER_EMAIL_ADDRESS": "Enter your email address",
   "CLI_AUTH_LOGIN_ENTER_PASSWORD": "Enter your password",
   "CLI_AUTH_LOGIN_SUCCESS": "Successfully logged in!!",
-  "CLI_AUTH_LOGIN_FAILED": "Login Error",
+  "CLI_AUTH_LOGIN_FAILED": "You need to login first. See: auth:login --help",
```

Comment:
Hi @akashbhosale97,
This message CLI_AUTH_LOGIN_FAILED is also used in login command's catch handler, this message will not be appropriate there.